packages/cascara/pages/30-structures-and-layouts.mdx

```diff
@@ -37,6 +37,13 @@ Examples of functional layouts:
 - Chat
 - List View User Directory
 
+<Rule
+  title='A Rule'
+  body='Do not do any bad things here or else we will cut you.'
+  doImg='https://placedog.net/500/500?id=1'
+  dontImg='https://placedog.net/500/500?id=2'
+/>
```

Comment:
This is going to get changed before the release when I take Allie through the docs on Thursday.